bin/is-website-vulnerable.js

```diff
@@ -5,65 +5,78 @@
 const os = require('os')
 const debug = require('debug')('is-website-vulnerable')
 const argv = require('yargs').argv
+const { prompt } = require('enquirer')
 const { Audit, RenderConsole, RenderJson, Utils } = require('../index')
 
-let url = process.argv[2]
-debug(`received url argument: ${url}`)
+const main = async () => {
+  let url = process.argv[2]
+  debug(`received url argument: ${url}`)
 
-url = Utils.parseUrl(url)
+  const isWindows = os.type() === 'Windows_NT'
+  const isJson = !!argv.json
+  const showProgressBar = !isJson && !isWindows
 
-if (!url) {
-  console.error('')
-  console.error('error: please provide a URL of a website to scan')
-  console.error('')
-  console.error('Usage:')
-  console.error('  is-website-vulnerable https://www.google.com')
-  console.error('')
-  process.exit(1)
-}
+  if (!url) {
+    if (isJson) {
+      console.error('')
+      console.error('Error: please provide a URL of a website to scan.')
```

Comment:
We should probably simply throw an `Error` here, and let Node to its job.

Reply:
The only issue is that it will be formatted as an actual error object with stacktrace and all and I'd rather control the output that gets shown on an error

Reply:
Could you take another look at this? I updated the main method to properly log errors.